docs/en-US/component/tooltip.md

```diff
@@ -157,6 +157,7 @@ tooltip/animations
 | content                   | display content, can be overridden by `slot#content`                                                                                                    | ^[string]                                                                                                                                                                   | ''                |
 | raw-content               | whether `content` is treated as HTML string                                                                                                             | ^[boolean]                                                                                                                                                                  | false             |
 | placement                 | position of Tooltip                                                                                                                                     | ^[enum]`'top' \| 'top-start' \| 'top-end' \| 'bottom' \| 'bottom-start' \| 'bottom-end' \| 'left' \| 'left-start' \| 'left-end' \| 'right' \| 'right-start' \| 'right-end'` | bottom            |
+| fallback-placements       | list of possible positions for Tooltip [popper.js](https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements)                                    | ^[arrary]`Placement[]`                                                                                                                                                      | —                 |
```

Comment:
Does this need version tag?

Reply:
It shouldn't be necessary. It already supports it, but the documentation is not written.

Reply:
Alright